src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo, onClose,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    const getUserFromApi = async () => {
+      const response = await getUser(selectedTodo.userId);
+
+      setUser(response);
+    };
+
+    getUserFromApi();
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!user?.id ? (
         <Loader />
       ) : (
```

Comment:
A little more readable :)

```suggestion
      {!user?.id
        ? (
          <Loader />
        )
        : (
```